baybe/surrogates/gaussian_process/presets/default.py

```diff
@@ -0,0 +1,110 @@
+"""Default preset for Gaussian process surrogates."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from attrs import define
+
+from baybe.kernels.basic import MaternKernel, ScaleKernel
+from baybe.priors.basic import GammaPrior
+from baybe.surrogates.gaussian_process.kernel_factory import KernelFactory
+
+if TYPE_CHECKING:
+    from torch import Tensor
+
+    from baybe.kernels.base import Kernel
+    from baybe.searchspace.core import SearchSpace
+
+
+@define
+class DefaultKernelFactory(KernelFactory):
```

Comment:
I'd prefer if you call all of the factories in here already `EDBO...` and not `Default`

Reply:
Perhaps this a slight misunderstanding: The stuff in "default" is what I intend to be OUR setting (which is also the default used in the GP class). The subtlety: the code already differs from EDBO in the one line we changed (and which I highlighted via comment). Therefore, the docstring also says "adapted from EDBO" instead of just "EDBO". I know this is only a minor modification, but not calling it EDBO allows us to freely modify the content of this file. I'd rather have a separate "EDBO" preset that *exactly* mirrors the EDBO mechanism and will never be touched again in the future. Does it make sense to you?

Reply:
ok makes sense, in that case though why is there no edbo factory included?

Reply:
No prob, can add. I just wanted to push the PR draft first so that we can flesh out all details before I add unnecessary code that needs to be reviewed / thrown away again. Will add once we've clarified all open thread, just to be on the safe side. Will keep this thread open and ping you in the end once it's there

Reply:
as mentioned in the other threads, will add EDBO in the next PR since it requires likelihood extension